docs/source/explanation/additional_features/auto_configuration.rst

```diff
@@ -1,4 +1,49 @@
+
+    .. code-block::
+
+        $ otx build --train-data-root <path_to_data_root>
+
+Moreover, our dataset can have no train/val splits at all. The Datumaro manager integrated into OTX will handle it on its own.
+It will recognize the task by analizing the dataset and if there is no splits for the validation - Datumaro will do random auto-split, memorizing this split to further use it with ``otx eval`` and ``otx optimize``.
+
+.. note::
+
+    Currently, Datumaro auto-split feature supports 3 formats: `Imagenet <https://www.image-net.org/>`_  (multi-class classification), `COCO <https://cocodataset.org/#format-data>`_ (detection) and `Cityscapes <https://openvinotoolkit.github.io/datumaro/docs/formats/cityscapes/>`_ (semantic segmentation).
+
+After dataset preparation the training will be started with the middle-sized template to achieve a competitive accuracy preserving fast inference.
+
+.. note::
+
+    Supported dataset formats for each task:
+
+    - classificaiton: `Imagenet <https://www.image-net.org/>`_, `COCO <https://cocodataset.org/#format-data>`_ (multi-label), :ref:`custom hierarchical <hierarchical_dataset>`
+    - object detection: `COCO <https://cocodataset.org/#format-data>`_, `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_, `YOLO <https://openvinotoolkit.github.io/datumaro/docs/formats/yolo/>`_
+    - semantic segmentation: `Common Semantic Segmentation <https://openvinotoolkit.github.io/datumaro/docs/formats/common_semantic_segmentation/>`_, `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_, `Cityscapes <https://openvinotoolkit.github.io/datumaro/docs/formats/cityscapes/>`_, `ADE20k <https://openvinotoolkit.github.io/datumaro/docs/formats/ade20k2020/>`_
+    - action classification: `CVAT <https://opencv.github.io/cvat/docs/manual/advanced/xml_format/>`_
+    - action : `CVAT <https://opencv.github.io/cvat/docs/manual/advanced/xml_format/>`_
+    - anomaly classificaiton: `MVTec <https://www.mvtec.com/company/research/datasets/mvtec-ad>`_
+    - anomaly detection: `MVTec <https://www.mvtec.com/company/research/datasets/mvtec-ad>`_
+    - anomaly segmentation: `MVTec <https://www.mvtec.com/company/research/datasets/mvtec-ad>`_
+    - instance segmentation: `COCO <https://cocodataset.org/#format-data>`_, `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_
+
+    If we have a dataset format occluded with other tasks, for example ``COCO`` format, we should directly emphasize the task type with an additional CLI option. If not, OTX automatically chooses the task type that you might don't intend:
+
+    .. code-block::
+
+        $ otx train --train-data-root <path_to_data_root> --task {CLASSIFICATION, DETECTION, SEGMENTATION, ACTION_CLASSIFICATION, ACTION_DETECTION, ANOMALY_CLASSIFICATION, ANOMALY_DETECTION, ANOMALY_SEGMENTATION, INSTANCE_SEGMENTATION}
```

Comment:
Current `build` and `train` CLI have different arguments,

For the `otx build` --> `--model`, `--backbone`, `--train_type`, `--task`, ...
For the `otx train` --> `--train/val-data-roots`, `--template`, `--save-model-to`, ...

`otx build --train-data-root {}` --> OK.
`otx build --train-data-root {} --task {}` --> OK.
`otx build --train-data-root {} --template {}` --> Can't

`otx train --train-data-root {}` --> OK.
`otx train --train-data-root {} --task {}` --> Can't
`otx train --train-data-root {} --template {}` --> OK.

Reply:
I changed `train` to `build` in case when we have occluded datasets